### DISCUSSION TITLE
Loading time is very long

### DISCUSSION THREAD

**Discussion by HarryWangATX** [auroral-ui/hexo-theme-aurora#51] [Q&A] [answered]
The first time loading the site takes over 10 seconds.

**Comment by bennyxguo** [comment, marked as answer]
@HarryWangATX, this kind of "question" type problem, please first post it to the Q&A section in the discussion of this project. 

Please check if you used the international CDN.

**Comment by HarryWangATX** [comment]
okay thank you! i switched the CDN and its way faster now
